src/omni-slider.js

```diff
@@ -202,6 +204,21 @@ Slider.prototype.init = function(options) {
   }
 };
 
+Slider.prototype._standardizeUSD_ = function(number) {
+
+  var numberinString = number.toString();
```

Comment:
do you think its worth going es6 on these new features since babel is applied but not used? (i.e. using const instead of var)

also part 2: should we make sure the `number` arg passed is indeed a number?